addons/website_event/controllers/main.py

```diff
@@ -215,18 +217,25 @@ def _prepare_event_register_values(self, event, **post):
         }
 
     @http.route('/event/add_event', type='json', auth="user", methods=['POST'], website=True)
-    def add_event(self, event_name="New Event", **kwargs):
-        event = self._add_event(event_name, request.context)
+    def add_event(self, event_name, event_start, event_end, address_id, **kwargs):
+        values = {
+            'name': event_name,
+            'date_begin': event_start,
+            'date_end': event_end,
+            'address_id': address_id
+        }
+        event = self._add_event(values, request.context)
         return "/event/%s/register?enable_editor=1" % slug(event)
 
-    def _add_event(self, event_name=None, context=None, **kwargs):
-        if not event_name:
-            event_name = _("New Event")
-        date_begin = datetime.today() + timedelta(days=(14))
+    def _add_event(self, values, context=None, **kwargs):
```

Comment:
While we are at it, can't we get rid of that context=None brol ? Seems it is used for a quite-poopy context key to add default ticket in website_event_sale. I bet we could clean that brol completely.